client/components/app.jsx

```diff
@@ -274,7 +293,9 @@ class App extends React.Component {
             </SecondStar>
```

Comment:
Is this a remnant? You have a close `</SecondStar>` tag but not a corresponding open tag.